doc/api/events.markdown

```diff
@@ -137,10 +137,17 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+* `type` {String} The type of event
```

Comment:
Oops, sorry. There is no clear way to know if there are multiple updates in the same PR. I missed that comment :( I'll update it now.